libs/partners/openai/langchain_openai/chat_models/base.py

```diff
@@ -475,30 +475,40 @@ def _stream(
     ) -> Iterator[ChatGenerationChunk]:
         message_dicts, params = self._create_message_dicts(messages, stop)
         params = {**params, **kwargs, "stream": True}
+        if stream_options := kwargs.get("stream_options"):
+            params["stream_options"] = stream_options
```

Comment:
is this not already handled by model_kwargs?

Reply:
I removed these lines as they're actually unnecessary. Either of these will work:
```python
model = ChatOpenAI(model_kwargs={"stream_options": {"include_usage": True}})
response = model.stream("hello")
```

```python
model = ChatOpenAI()
response = model.stream("hello", stream_options={"include_usage": True})
```

For clarity, the effect of setting `True` there is to add a new, empty AIMessage to the stream containing token usage in the response metadata.

Without parameter:
```
[AIMessageChunk(content='', id='run-45ea65c3-ebdc-4bff-b84f-c3107999a86d'),
 AIMessageChunk(content='Hello', id='run-45ea65c3-ebdc-4bff-b84f-c3107999a86d'),
 AIMessageChunk(content='!', id='run-45ea65c3-ebdc-4bff-b84f-c3107999a86d'),
 AIMessageChunk(content='', response_metadata={'finish_reason': 'stop'}, id='run-45ea65c3-ebdc-4bff-b84f-c3107999a86d')]
```
With parameter:
```
[AIMessageChunk(content='', id='run-eeb6f9ed-0255-4aaf-8ead-f39b1d896b6a'),
 AIMessageChunk(content='Hello', id='run-eeb6f9ed-0255-4aaf-8ead-f39b1d896b6a'),
 AIMessageChunk(content='!', id='run-eeb6f9ed-0255-4aaf-8ead-f39b1d896b6a'),
 AIMessageChunk(content='', response_metadata={'finish_reason': 'stop'}, id='run-eeb6f9ed-0255-4aaf-8ead-f39b1d896b6a'),
 AIMessageChunk(content='', response_metadata={'token_usage': {'completion_tokens': 9, 'prompt_tokens': 8, 'total_tokens': 17}}, id='run-eeb6f9ed-0255-4aaf-8ead-f39b1d896b6a')]
```